compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/AnnotationDesugar.java

```diff
@@ -300,6 +300,10 @@ private void attachSchedulerPolicy(BLangFunction function) {
             if (!annotation.annotationName.value.equals("strand")) {
                 continue;
             }
+            if (annotation.expr == null) {
```

Comment:
Shouldn't this get added at desugar?

I'm assuming we're adding this here because of the field-wise check after this? IMO, we should fix that check instead of adding an empty mapping constructor expression here.